packages/api/helper/googleDrive.js

```diff
@@ -31,6 +32,8 @@ function createFolder(applicationName) {
       console.error(err);
     }
   })
+
+  console.log(`folder has been created for ${applicationName}!`);
```

Comment:
Nitpick: Debugging `console.log`s should be removed.

Reply:
I thought of this more like an indication that the app is working — there are a few "email has sent" console.logs that have been helpful in the dev process. But can remove this if it's not necessary.